services/localfile/client/client.go

```diff
@@ -761,18 +761,26 @@ func (c *chmodCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interfa
 		fmt.Fprintln(os.Stderr, "please specify a filename to chmod")
 		return subcommands.ExitUsageError
 	}
-	if c.mode == 0 {
-		fmt.Fprintln(os.Stderr, "--mode must be set to a non-zero value")
+	if c.mode == "" {
+		fmt.Fprintln(os.Stderr, "--mode must be set to a non-empty value")
 		return subcommands.ExitFailure
 	}
 
+	mode, err := ParseFileMode(c.mode)
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Invalid --mode %s", c.mode)
```

Comment:
I think we should get a newline here at the end of message, when I pulled and tested this ended up with output:

```
mwalas@ % go run ./cmd/sanssh --targets=localhost file chmod --mode=898 /tmp/test

Invalid --mode 898exit status 1
% 
```

Reply:
Good catch. I added a new line.